src/log.js

```diff
@@ -71,9 +71,9 @@ export class Log {
   /**
    * @param {!Window} win
    * @param {function(!./mode.ModeDef):!LogLevel} levelFunc
-   * @param {string=} opt_suffix
+   * @param {boolean=} opt_userError
```

Comment:
I see you pass in opt_userError in both user and dev logConstructor(). Is this still optional? or we want to make it a required value? 
nit: would `opt_isUserError` better?